tests/python_compiler/transforms/test_xdsl_merge_rotations.py

```diff
@@ -251,17 +193,31 @@ def test_controlled_composable_ops_different_control_values(self, run_filecheck)
             }
         """
 
-        ctx = Context()
-        ctx.load_dialect(arith.Arith)
-        ctx.load_dialect(func.Func)
-        ctx.load_dialect(test.Test)
-        ctx.load_dialect(Quantum)
+        pipeline = (MergeRotationsPass(),)
+        run_filecheck(program, pipeline)
+
+
+# pylint: disable=too-few-public-methods
+@pytest.mark.usefixtures("enable_disable_plxpr")
+class TestMergeRotationsIntegration:
+    """Integration tests for the MergeRotationsPass."""
+
+    def test_qjit(self, run_filecheck_qjit):
+        """Test that the MergeRotationsPass works correctly with qjit."""
+        dev = qml.device("lightning.qubit", wires=2)
```

Comment:
I believe it would be beneficial to add this example to the `docstr` of `run_filecheck_qjit()`

Reply:
Added.